tests/Integration/Decoder.elm

```diff
@@ -144,4 +163,7 @@ all =
         , test "should decode Rules" <|
             always <|
                 Expect.equal (Ok expectedRules) (decodeString decodeRules rulesJson)
+        , test "should decode labels" <|
+            always <|
+                Expect.equal (Ok expectedLabels) (decodeString decodeLabels labelsJson)
```

Comment:
A test case for an empty dict and a more complex one (object or ref) would be nice.